spec/acceptance/should_create_task_spec.rb

```diff
@@ -4,6 +4,14 @@
 
 describe "Should create a scheduled task", :node => host do
 
+  before(:all) do
+    @username, @password = add_test_user(host)
+  end
```

Comment:
Why don't you move the secondary user creation up to here as well... then you can be sure that you remove it in the `after`, rather than inlining deletions that might never occur.